### ISSUE TITLE
Error: Plugin Disabler only works as a must-use plugin in a bedrock site.

### ISSUE THREAD

**Issue by GiladEhven** [lukasbesch/bedrock-plugin-disabler#15]
Bedrock: `1.20.0`
Trellis: `1.15.0`
Trellis CLI: `1.8.0` (project created with `1.7.0`)

Plugin installed with `composer require lukasbesch/bedrock-plugin-disabler`.

The following has been added to `site/config/environments/remote.php` (not a Production or Staging droplet, but a custom remote that works in every respect):

```
Config::define('DISABLED_PLUGINS', [
    'wp-cloudflare-page-cache/wp-cloudflare-super-page-cache.php',
    'wp-rocket/wp-rocket.php'
]);
```

The `bedrock-plugin-disabler` plugin installs as expected in Dev (showing up in the `mu-plugins` directory).

Deploying to a remote (provisioned via Trellis in DigitalOcean), I get:

```
TASK [deploy : Invoke 'wp core is-installed' command] **************************
System info:
  Ansible 2.12.9; Darwin
  Trellis version (per changelog): "Add built-in fail2ban filters"
---------------------------------------------------
non-zero return code
Error: Plugin Disabler only works as a must-use plugin in a bedrock site.
fatal: [example_com_REMOTE]: FAILED! => {"changed": false, "cmd": ["wp", "core", "is-installed", "--skip-plugins", "--skip-themes"], "delta": "0:00:00.719287", "end": "2023-02-02 04:35:29.243197", "failed_when_result": true, "rc": 1, "start": "2023-02-02 04:35:28.523910", "stderr_lines": ["Error: Plugin Disabler only works as a must-use plugin in a bedrock site."], "stdout": "", "stdout_lines": []}

PLAY RECAP *********************************************************************
example_com_REMOTE         : ok=25   changed=9    unreachable=0    failed=1    skipped=32   rescued=0    ignored=0   
localhost                  : ok=0    changed=0    unreachable=0    failed=0    skipped=1    rescued=0    ignored=0   

exit status 2
```

This same remote accepted many deployments before adding this plugin, and removing the plugin from `composer.json` restores deployability.

**Comment by lukasbesch** [comment]
Thanks for the report!

Apparently, plugin activation hooks of mu-plugins are run every time you use a WP-CLI command.
It is now only registered when the plugin detected it is not a must-use plugin.

Can you confirm if this works for you? Fix is on the master branch, will release a new version later.

**Comment by GiladEhven** [comment]
Fix confirmed, thank you!

And thank you for explaining the underlying issue.

Do you have an ETA for the next release?

**Comment by lukasbesch** [comment]
Released in [v1.4.0](https://github.com/lukasbesch/bedrock-plugin-disabler/releases/tag/v1.4.0), thanks @GiladEhven
